parasoft-findings-teamcity-agent/src/main/resources/com/parasoft/findings/teamcity/agent/sa-pmdcpd.xsl

```diff
@@ -47,10 +47,10 @@
             </xsl:if>
             <xsl:if test="@srcRngFile">
                 <xsl:attribute name="path">
-                    <xsl:if test="/ResultsSession/@toolId = 'jtest'">
+                    <xsl:if test="../../@lang = 'java'">
```

Comment:
toolId on <ResultsSession> tag is optional but lang on <DupViol> tag is required.

Reply:
Is there any reason to use "../../" instead of selecting a child element of "StdViols"?